core/shared/src/main/scala-3/me/shadaj/scalapy/py/Facades.scala

```diff
@@ -0,0 +1,21 @@
+package me.shadaj.scalapy.py
+
+import scala.language.dynamics
+
+import me.shadaj.scalapy.interpreter.PyValue
+
+trait StaticModule(name: String) extends Module:
```

Comment:
Let's stick to curly-brace syntax for now, so that things are more uniform across Scala 2/3